src/ISA/ISA/ArcTypes/ArcTable.fs

```diff
@@ -621,25 +615,30 @@ type ArcTable =
         ]
         |> String.concat "\n"
 
+    member this.StructurallyEquivalent (other: ArcTable) =
```

Comment:
Now that I think about it. Maybe rename to "StructurallyEquals" for consistent naming?